### DISCUSSION TITLE
iOS/iPadOS osu!lazer issues

### DISCUSSION THREAD

**Discussion by nxfx21** [ppy/osu#16050] [Q&A] [answered]
My first issue: osu! needs to be updated to work on iOS/iPadOS because it doesn't open, it says that the [developer](https://github.com/ppy) needs to update it.
My second issue: osu!lazer builds should be also released as *.ipa files.
Resolve these issues and osu!lazer on mobile platforms is perfect.

**Comment by peppy** [comment, marked as answer]
You can't load ipa files. That's not how iphones work.

Your OS is likely out of date if you are seeing that message.

**Reply by nxfx21** [reply]
uhh ppy, I'm using ios 15.1 on my iPad gen 7, and you can drag ipa files into itunes. or install them with altstore

**Reply by peppy** [reply]
We don't support anything that isn't the testflight install. Sorry.
